lte/gateway/python/scripts/pipelined_cli.py

```diff
@@ -65,6 +65,114 @@
 )
 
 
+@grpc_wrapper
+def stress_test5g(client, args):
+    """
+    attach/detach script for smf session for 5g stress test
+    """
+    ue_dict = _gen_ue_set(args.number_of_iterations)
+    session_id = 100
+    version = 2
+    apn_flag = True
+    delta_time = 1 / args.attaches_per_sec
+    print("Attach every ~{0} seconds".format(delta_time))
+
+    if args.disable_qos:
+        print("QOS Disabled")
+        apn_flag = False
+    else:
+        print("QOS Enabled")
+        apn_flag = True
+
+    in_teid = 50
+    out_teid = 100
+    print("Starting attaches")
+    timestamp = datetime.now()
+    completed_reqs = 0
+    for ue in ue_dict:
+        grpc_start_timestamp = datetime.now()
+        cls_sess = CreateSessionUtil(ue.imsi_str, session_id, version)
+        cls_sess.CreateSession(
+            ue.imsi_str, 'ADD', in_teid, out_teid,
+            ue.ipv4_src, ue.ipv4_dst,
+            '', ue.rule_id, '',
+            'allow', 10, apn_flag,
+        )
+        client.SetSMFSessions(cls_sess._set_session)
+        grpc_end_timestamp = datetime.now()
+        call_duration = (grpc_end_timestamp - grpc_start_timestamp).total_seconds()
+        if call_duration < delta_time:
+            time.sleep(delta_time - call_duration)
+        if completed_reqs % LOG_INCREMENT == 0:
+            print("Finished {0}".format(completed_reqs))
+        completed_reqs += 1
+        in_teid += 1
+        out_teid += 1
+
+    duration = (datetime.now() - timestamp).total_seconds()
+    print("Finished {0} attaches in {1} seconds".format(len(ue_dict), duration))
+    print("Actual attach rate = {0} UEs per sec".format(round(len(ue_dict) / duration)))
+
+    # detach script
+    in_teid = 50
+    out_teid = 100
+    time.sleep(args.time_between_detach)
+    print("Starting detaches")
+    timestamp = datetime.now()
+    completed_reqs = 0
+    for ue in ue_dict:
+        grpc_start_timestamp = datetime.now()
+        cls_sess = CreateSessionUtil(ue.imsi_str, session_id, version)
+        cls_sess.CreateSession(
+            ue.imsi_str, 'REMOVE', in_teid, out_teid,
+            ue.ipv4_src, ue.ipv4_dst,
+            ue.rule_id, '', '',
+            'allow', 10,
+        )
+        client.SetSMFSessions(cls_sess._set_session)
+        grpc_end_timestamp = datetime.now()
+        call_duration = (grpc_end_timestamp - grpc_start_timestamp).total_seconds()
+        if call_duration < delta_time:
+            time.sleep(delta_time - call_duration)
+        if completed_reqs % LOG_INCREMENT == 0:
+            print("Finished {0}".format(completed_reqs))
+        completed_reqs += 1
+        in_teid += 1
```

Comment:
this doesn't look clean, lets move the teid into the UEInfo

Reply:
fixed